core/quantity/src/Parser.ts

```diff
@@ -796,6 +800,94 @@ export class Parser {
     return { ok: true, value: magnitude };
   }
 
+  private static parseRatioFormat(inString: string, spec: ParserSpec): QuantityParseResult {
+    if (!inString || !inString.includes(":"))
+      return { ok: false, error: ParseError.NoValueOrUnitFoundInString };
+
+    const parts = inString.split(":");
+    if (parts.length !== 2)
+      throw new QuantityError(QuantityStatus.UnableToConvertParseTokensToQuantity, `Invalid ratio: ${inString}, needs to be in form x:y`);
```

Comment:
I say an invalid input string is not an exception (unexpected). It's user input, so it can be malformed. Also, if the input string is just a single number, like "30", we may interpret that as "30:1". I reckon that's what a user would expect for that input